pixeltable/exec/data_row_batch.py

```diff
@@ -90,7 +90,8 @@ def flush_imgs(
             idx_range = slice(0, len(self.rows))
         for row in self.rows[idx_range]:
             for info in stored_img_info:
-                filepath = str(MediaStore.prepare_media_path(self.tbl.id, info.col.id, self.tbl.get().version))
+                col = info.col
+                filepath = str(MediaStore.prepare_media_path(col.tbl.id, col.id, col.tbl.version))
```

Comment:
What are the implications of using `col.tbl.id` instead of `self.tbl.id`?

I'd feel more comfortable with this change if we put in an

`assert col.tbl.id == self.tbl.id`

(and if that assertion fails I'd want to understand why).